### ISSUE TITLE
Add complete parts/vitamin list to first page of each instruction set

### ISSUE THREAD

**Issue by natecampbell-zz** [machineagency/jubilee#105]
It would be great if the first page of each instruction set (cross bar, corner plates, ect.) Included a complete list of all parts and fasteners needed for that task

**Comment by Poofjunior** [comment]
Yeah; this would be great, and doesn't require that much bookkeeping on the software side to maintain. It also seems like it would help folks decide up-front whether they can build certain subsections while they're waiting on other bits to ship.
